RiotSwiftUI/Modules/Authentication/ForgotPassword/View/AuthenticationForgotPasswordScreen.swift

```diff
@@ -0,0 +1,147 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct AuthenticationForgotPasswordScreen: View {
+
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme
+    
+    // MARK: Public
+    
+    @ObservedObject var viewModel: AuthenticationForgotPasswordViewModel.Context
+    
+    // MARK: Views
+    
+    var body: some View {
+        GeometryReader { geometry in
+            VStack {
+                ScrollView {
+                    mainContent
+                        .readableFrame()
+                        .padding(.horizontal, 16)
+                }
+                
+                if viewModel.viewState.hasSentEmail {
+                    waitingFooter
+                        .padding(.bottom, geometry.safeAreaInsets.bottom > 0 ? 20 : 36)
```

Comment:
Could this bottom padding for the buttons match the padding on other breaker screens like so:
https://github.com/vector-im/element-ios/blob/62add618b310f9efd92da04d1ddc1ba64c08bdeb/RiotSwiftUI/Modules/Onboarding/Congratulations/View/OnboardingCongratulationsScreen.swift#L54-L55